client/ayon_core/plugins/publish/cleanup.py

```diff
@@ -71,10 +72,12 @@ def process(self, instance):
             self.log.debug("Cleaning renders new...")
             self.clean_renders(instance, skip_cleanup_filepaths)
 
-        if [ef for ef in self.exclude_families
-                if instance.data["productType"] in ef]:
+        product_type = instance.data["productType"]
```

Comment:
Oh lord, this plugin...

This condition should probably happen before error checks. And we should change `exclude_families` to `exclude_product_types`.

I don't know why there was "str in str" check instead of "str == str"? I guess that is question for @jakubjezek001 who might be able to tell if there are product types that should be skipped too and contain `"clip`"?

Reply:
I don't remember it anymore.. Its been reeeealy old code. But there could be case, or rather used to be, where **render** in nuke was **render.local**, **render.farm** and so on..

Reply:
So it should be safe to replace with explicit product type check...?

Reply:
> So it should be safe to replace with explicit product type check...?

I am just guessing but it might be a solution.

Reply:
This is all that's keeping this from getting merged? Should I revert to behavior like before this PR?

Reply:
Just pinging... what to do?

Reply:
> This is all that's keeping this from getting merged? Should I revert to behavior like before this PR?

Probably yes. It is changing behavior that looks like is unpredictable at this moment.